src/diffusers/models/autoencoder_tiny.py

```diff
@@ -334,7 +331,7 @@ def forward(
         """
         enc = self.encode(sample).latents
         scaled_enc = self.scale_latents(enc).mul_(255).round_().byte()
-        unscaled_enc = self.unscale_latents(scaled_enc)
+        unscaled_enc = self.unscale_latents(scaled_enc / 255.0)
```

Comment:
Worth adding a comment on why we're scaling here since our pixel normalization / de-normalization logic is handled in the `image_processor.py` module.

Reply:
The `/ 255.0` here is necessary to invert the `.mul_(255).round_().byte()` on the preceding line. In https://github.com/madebyollin/taesd/blob/main/taesd.py#L86, the division by 255 is [handled](https://github.com/pytorch/vision/blob/main/torchvision/transforms/functional.py#L174) automatically by torchvision's `to_tensor`, but this code doesn't use `to_tensor` so we need to apply the division ourselves.

Reply:
Ah cool. Maybe worth adding a comment explaining it :)

Reply:
Done, hopefully it's clearer now